xlator/options.go

```diff
@@ -1,26 +1,32 @@
 package xlator
 
+import (
+	"errors"
+	"strconv"
+	"strings"
+)
+
 // OptionType is a type which represents the type of xlator option.
 type OptionType int
 
 // These are all the possile values for the type OptionType.
 const (
 	OptionTypeAny OptionType = iota
-	OptionTypeStr
-	OptionTypeInt
-	OptionTypeSizet
-	OptionTypePercent
-	OptionTypePercentOrSizet
 	OptionTypeBool
-	OptionTypeXlator
-	OptionTypePath
-	OptionTypeTime
+	OptionTypeClientAuthAddr
```

Comment:
The order here needs to match the order of the `volume_option_type_t` enum in options.h. Don't sort it alphabetically.

Reply:
oops, I wasn't aware of that.
Ack, will do.